### DISCUSSION TITLE
Are you open to design contributions?

### DISCUSSION THREAD

**Discussion by fivenp** [mifi/lossless-cut#1394] [Q&A]
Hey there!

I just discovered LoslessCut and I really enjoyed trying it out.... Now through the nature of being a designer and somewhat an engineer loving OSS I was curious if you'd be open to design contributions?

Talking to a lot of other designers in the past there seems to be a huge interest in them contributing to OSS projects – they just struggle to understand how to do so.

Therefor I've built https://contribute.design/ which helps designers and open source software to understand each other better. Basically I'm trying to help projects get the guidelines for designers set up and create some visibility across designers for the project.

My question now is – would you be open to participate in this initiative and can I potentially support you in any way here? I'd really love to enable other designers to participate in amazing OSS projects....

**Comment by mifi** [comment]
Hi. PRs are certainly welcome for improving design. Not sure what’s the value of creating design guidelines when the app isn’t currently following any. But if someone wants to make some  contributions in regards to making the UI/coloring etc more consistent then I welcome that. If u want to add llc to your site you can do that

**Reply by fivenp** [reply]
Amazing to hear :) 
The guidelines would be more targeted towards having a rough introduction for designers (e.g. here's how you can interact with the project, here are some existing visuals, etc) and then having designers extending these docs over time. 

I'll try to prep a PR for you to review to get a better feeling for it :)
